react/game/src/app/components/PrivateRoute/index.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import { Route, Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
 
-const PrivateRoute = ({ component: Component, authenticated, ...rest }) => (
-  <Route
-    {...rest}
-    render={props => (authenticated === true ? <Component {...props} /> : <Redirect to="/" />)}
-  />
-);
+const PrivateRoute = ({ component: Component, loggedin, ...rest }) => {
+  const res = props => (loggedin ? <Component {...props} /> : <Redirect to="/" />);
```

Comment:
In the next pr don't forget to extract this functions as a prop of a class component